www/docs/en/dev/guide/platforms/ios/webview.md

```diff
@@ -220,5 +222,16 @@ After using either of these two methods, continue with the **"Using CDVViewContr
     */
     viewController.wwwFolderName = @"myfolder";
     viewController.startPage = @"mypage.html"
-    ```
+    ```
+## Adding Plugins files used in you cordova project
```

Comment:
When user want to integrate cordova project which used plugins in xcode project.  They also need to add source code of these plugins into their xcode project. Otherwise plugins will not work when use cordova  CDVViewController in xcode  project.